instrumentation/build.gradle

```diff
@@ -31,7 +31,6 @@ dependencies {
 
 android {
     compileSdkVersion COMPILE_SDK_VERSION as int
-    buildToolsVersion BUILD_TOOLS_VERSION as String
```

Comment:
Build tools declarations are no longer required and Tor said they should be removed.